### DISCUSSION TITLE
Traffic Information

### DISCUSSION THREAD

**Discussion by umpfmpf** [devemux86/cruiser#618] [Q&A] [answered]
Would it be possible to include traffic information, similar to what Sygic does, for example to access TomTom traffic and use it in navigation and planning?

**Comment by devemux86** [comment, marked as answer]
@umpfmpf thanks for the suggestion.

Traffic data are not free or open source like OpenStreetMap,
so we have to see if they are provided to other apps and under what conditions.

**Reply by umpfmpf** [reply]
At Sygic it costs around €16 per year. 
It might be an option to introduce an extended PRO version with an adjusted annual fee, so that there would then be a multi-stage subscription system.
